handlers/bounty_test.go

```diff
@@ -563,3 +594,183 @@ func TestGetOrganizationPreviousBountyByCreated(t *testing.T) {
+		req, err := http.NewRequest("GET", "/wanteds/assigned/uuid", nil)
+		req = req.WithContext(ctx)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		bHandler.GetPersonAssignedBounties(rr, req)
+
+		assert.Equal(t, http.StatusOK, rr.Code)
+
+		var responseData []db.BountyResponse
+		err = json.Unmarshal(rr.Body.Bytes(), &responseData)
+		if err != nil {
+			t.Fatalf("Error decoding JSON response: %s", err)
+		}
+
+		assert.Empty(t, responseData)
+		assert.Len(t, responseData, 0)
+	})
+}
+
+func TestGetPersonCreatedBounties(t *testing.T) {
+	ctx := context.Background()
+	mockDb := dbMocks.NewDatabase(t)
+	mockHttpClient := mocks.NewHttpClient(t)
+	bHandler := NewBountyHandler(mockHttpClient, mockDb)
+
+	t.Run("should return bounties created by the user", func(t *testing.T) {
+		mockGenerateBountyResponse := func(bounties []db.Bounty) []db.BountyResponse {
+			var bountyResponses []db.BountyResponse
+
+			for _, bounty := range bounties {
+				owner := db.Person{
+					ID: 1,
+				}
+				assignee := db.Person{
+					ID: 1,
+				}
+				organization := db.OrganizationShort{
+					Uuid: "uuid",
+				}
+
+				bountyResponse := db.BountyResponse{
+					Bounty:       bounty,
+					Assignee:     assignee,
+					Owner:        owner,
+					Organization: organization,
+				}
+				bountyResponses = append(bountyResponses, bountyResponse)
+			}
+
+			return bountyResponses
+		}
+		bHandler.generateBountyResponse = mockGenerateBountyResponse
+
+		mockDb.On("GetCreatedBounties", mock.Anything).Return([]db.Bounty{
+			{ID: 1, OwnerID: "user1"},
+			{ID: 2, OwnerID: "user1"},
+		}, nil).Once()
+
+		rr := httptest.NewRecorder()
+		req, err := http.NewRequest("GET", "/wanteds/created/uuid", nil)
+		req = req.WithContext(ctx)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		bHandler.GetPersonCreatedBounties(rr, req)
+
+		assert.Equal(t, http.StatusOK, rr.Code)
+
+		var responseData []db.BountyResponse
+		err = json.Unmarshal(rr.Body.Bytes(), &responseData)
+		if err != nil {
+			t.Fatalf("Error decoding JSON response: %s", err)
+		}
+
+		assert.NotEmpty(t, responseData)
+		assert.Len(t, responseData, 2)
+	})
+
+	t.Run("should not return bounties created by other users", func(t *testing.T) {
+		mockGenerateBountyResponse := func(bounties []db.Bounty) []db.BountyResponse {
+			return []db.BountyResponse{}
+		}
+		bHandler.generateBountyResponse = mockGenerateBountyResponse
+
+		mockDb.On("GetCreatedBounties", mock.Anything).Return([]db.Bounty{}, nil).Once()
+
+		rr := httptest.NewRecorder()
+		req, err := http.NewRequest("GET", "/wanteds/created/uuid", nil)
+		req = req.WithContext(ctx)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		bHandler.GetPersonCreatedBounties(rr, req)
+
+		assert.Equal(t, http.StatusOK, rr.Code)
+
+		var responseData []db.BountyResponse
+		err = json.Unmarshal(rr.Body.Bytes(), &responseData)
+		if err != nil {
+			t.Fatalf("Error decoding JSON response: %s", err)
+		}
+
+		assert.Empty(t, responseData)
+		assert.Len(t, responseData, 0)
```

Comment:
same with this assertion

Reply:
I think there is not need as the response data is already empty